APMrover2/Steering.cpp

```diff
@@ -223,6 +227,10 @@ void Rover::set_servos(void) {
         if (failsafe.bits & FAILSAFE_EVENT_THROTTLE) {
             // suppress throttle if in failsafe and manual
             channel_throttle->set_radio_out(channel_throttle->get_radio_trim());
+            // suppress steer if in failsafe and manual and skid steer mode
+            if (g.skid_steer_out) {
+                channel_steer->set_servo_out(channel_steer->get_radio_trim());
```

Comment:
Still not right.  This should be 
channel_steer->set_radio_out(channel_steer->get_radio_trim());
You can't mix the servo and radio settings.